contrib/opencensus-ext-datadog/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Changelog
 
-## Unreleased
-- Initial version.
+## 0.1.0
```

Comment:
This one hasn't been released to PyPI (e.g. https://pypi.org/project/opencensus/) yet. We should leave it as `Unreleased`.